### DISCUSSION TITLE
蓝灯根本无法使用

### DISCUSSION THREAD

**Discussion by l3218w8693h** [getlantern/lantern-forum-cn#1074] [General]
![Screenshot_20241208_095712_org_getlantern_lantern_MainActivity](https://github.com/user-attachments/assets/60415a19-bc8b-43a1-ac1a-3fc7ba798dc7)
问题也根本无法反馈申请退款！用了一个星期就只有几次连接成功！
![Screenshot_20241208_100132_com_eg_android_AlipayGphone_XRiverActivity](https://github.com/user-attachments/assets/3e375528-a983-436e-bd23-79cb78edca5c)
订单号2024112822001402091424169434

**Comment by MaxMKMax** [comment]
@l3218w8693h 您好. 请您提供我账号邮箱, 谢谢您支持蓝灯.

**Reply by l3218w8693h** [reply]
邮箱是2466972431@qq.com
